flight/PiOS/Common/pios_mpu6000.c

```diff
@@ -51,6 +56,17 @@
 #endif
 #define MPU6000_SPI_LOW_SPEED               1000000
 
+// Private variables
```

Comment:
I suggest we conditionalize this on #ifndef SMALLF1 ... it's a lot of RAM and I doubt the little F1 targets can handle downsampling from high rates.